src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,284 @@
+    }
+
+    private static void printWelcomeMessage() {
+        printLine();
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        printLine();
+    }
+
+    private static void printByeMessage() {
+        printLine();
+        System.out.println("Bye. Hope to see you again soon! :)");
+        printLine();
+    }
+
+    public static void loadData() {
+        try {
+            File file = new File(FILE_PATH);
+            Scanner in = new Scanner(file);
+            while (in.hasNext()) {
+                loadTasks(in.nextLine());
+            }
+        } catch (FileNotFoundException e) {
+            File file = new File(FILE_PATH);
+            try {
+                file.createNewFile();
+            } catch (IOException ex) {
+                System.out.println("New file cannot be created");
+            }
+        }
+    }
+
+    public static void loadTasks(String input) {
+        String[] splitInput = input.split("->");
+        String taskType = splitInput[0].trim();
+        String status = splitInput[1].trim();
+        String taskDescription = splitInput[2].trim();
+
+        switch (taskType) {
+        case "T":
+            tasks.add(new Todo(taskDescription));
+            break;
+        case "D":
+            String by = splitInput[3].trim();
+            tasks.add(new Deadline(taskDescription, by));
+            break;
+        case "E":
+            String from = splitInput[3].trim();
+            String to = splitInput[4].trim();
+            tasks.add(new Event(taskDescription, from, to));
+            break;
+        }
+        if (status.equals("1")) {
+            tasks.get(tasks.size() - 1).markDone();
+        }
+    }
+
+    public static void saveTask() {
+        try {
+            FileWriter fw = new FileWriter(FILE_PATH);
+            for (Task task : tasks) {
+                String status;
+                String lineToAdd = "";
+                if (task.getStatusIcon().equals("X")) {
+                    status = "1";
+                } else {
+                    status = "0";
+                }
+                if (task instanceof Todo) {
+                    lineToAdd = "T" + " -> " + status + " -> " + task.description + "\n";
+                }
+                if (task instanceof Deadline) {
+                    lineToAdd = "D" + " -> " + status + " -> " + task.description + " -> " + ((Deadline) task).by + "\n";
+                }
+                if (task instanceof Event) {
+                    String time = ((Event) task).from + " -> " + ((Event) task).to;
+                    lineToAdd = "E" + " -> " + status + " -> " + task.description + " -> " + time + "\n";
+                }
+                fw.write(lineToAdd);
+            }
+            fw.close();
+        } catch (IOException e) {
+            System.out.println("Unable to save data");
+        }
+    }
```

Comment:
If you have time, you can consider to have a Storage class for these methods instead of put all of it in main class to improve code quality